src/main/java/org/minions/devfund/bruno/Squarelotron.java

```diff
@@ -0,0 +1,240 @@
+            System.arraycopy(squarelotron[index], 0, copy[index], 0, size);
+        }
+        return copy;
+    }
+
+    /**
+     * Update the index row with respective values.
+     *
+     * @param copy squarelotron matrix copy
+     * @param row  index row.
+     */
+    private void updateSquarelotron(final int[][] copy, final int row) {
+        final int colZero = 0;
+        final int colOne = 1;
+        final int colTwo = 2;
+        final int colThree = 3;
+        final int colFour = 4;
+        final int colFive = 5;
+        final int colSix = 6;
+        final int colSeven = 7;
+        final int rowThree = 3;
+        final int rowFour = 4;
+        final int rowFive = 5;
+        final int rowSix = 6;
+        final int rowSeven = 7;
+        int count = size - 1;
+        for (int col = 0; col < size; col++) {
+            switch (row) {
+                case 0:
+                    squarelotron[row][col] = copy[count][colZero];
+                    count--;
+                    break;
+                case 1:
+                    squarelotron[row][col] = copy[count][colOne];
+                    count--;
+                    break;
+                case 2:
+                    squarelotron[row][col] = copy[count][colTwo];
+                    count--;
+                    break;
+                case rowThree:
+                    squarelotron[row][col] = copy[count][colThree];
+                    count--;
+                    break;
+                case rowFour:
+                    squarelotron[row][col] = copy[count][colFour];
+                    count--;
+                    break;
+                case rowFive:
+                    squarelotron[row][col] = copy[count][colFive];
+                    count--;
+                    break;
+                case rowSix:
+                    squarelotron[row][col] = copy[count][colSix];
+                    count--;
+                    break;
+                case rowSeven:
+                    squarelotron[row][col] = copy[count][colSeven];
+                    count--;
+                    break;
+                default:
+                    System.out.println("error");
+                    break;
+            }
+        }
+    }
+
+    /**
+     * Gets the actual turns.
+     *
+     * @param numberOfTurns Integer that represent the moves number.
+     * @return the necessary turns.
+     */
+    private int getActualTurns(int numberOfTurns) {
+        final int squareSize = 4;
+        return numberOfTurns > 0 ? numberOfTurns % squareSize : squareSize - ((numberOfTurns % squareSize) * (-1));
+    }
+
+    /**
+     * Gets the squarelotron size.
+     *
+     * @return the squarelotron size.
+     */
+    public int getSize() {
+        return size;
+    }
+
+    /**
+     * Gets the squarelotron data.
+     *
+     * @return a sqarelotron matrix.
+     */
+    public int[][] getSquarelotron() {
+        final int[][] squarelotronCopy = squarelotron;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "squarelotronCopy". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)